starport/interface/cli/starport/cmd/network_proposal_approve.go

```diff
@@ -72,12 +72,16 @@ func networkProposalApproveHandler(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		verified, err := nb.VerifyProposals(cmd.Context(), chainID, home, ids, ioutil.Discard)
+		correct, reason, err := nb.VerifyProposals(cmd.Context(), chainID, home, ids, ioutil.Discard)
```

Comment:
I think, we should keep the return types as it was before, and even we don't need `correct` to be returned  here. `reason` should be an `error` type.

We can define a custom type error like below:

```go
type VerificationError struct {}
```

So it becomes to:
```suggestion
		err = nb.VerifyProposals(cmd.Context(), chainID, home, ids, ioutil.Discard)
```

Resulting with:

*
```go
// corresponds to `correct`
err == nil 
```

*
```go
var validationError *networkbuild.ValidationError
// corresponds to `reason`
errors.As(err, &validationError) == true

// otherwise corresponds to any other `err`
```

Reply:
Makes total sense, thx